cmd/kube-scheduler/app/options/options_test.go

```diff
@@ -218,7 +229,7 @@ profiles:
 	// multiple profiles config
```

Comment:
keep a test in place exercising multi-profile config with v1beta1

Reply:
Done.